code/wdb/datastripe.q

```diff
@@ -59,6 +59,15 @@ initdatastripe:{
 
 \d .ds
 
+createsymlink:{[tdpath;hdbpath;symfile]
+    /- function to create symlink to HDB sym file in taildir
+    tdsympath:1_raze string tdpath,"/",.proc.procname,"/",symfile;
+    hdbsympath:1_raze string hdbpath,"/",symfile;
+
+    /- linux command to create symlink in specified dirs
+    system"ln -s ",hdbsympath," ",tdsympath;
```

Comment:
I think we should error trap this and think about what we do in case of failure - how critical is the symlink, can we log an error and continue or do we need to do something more?

Reply:
Yep, I think logging an error and continuing would be fine - I'll add in some error trapping here 
I believe the symlink isn't critical to the functionality of the tailer - it's more to have a complete IDB/for the tail reader to have when it is loading data

Reply:
if the tail reader won't be able to load the intraday data without the symlink, it sounds pretty critical IMO - might be worth having a discussion on what should happen in the scenario that symlink fails